.gitignore

```diff
@@ -89,6 +89,7 @@ celerybeat-schedule
 .venv
 venv/
 ENV/
+Pipfile
```

Comment:
Observation: we may start using Pipfile in the future since this is the new standard.